composer.json

```diff
@@ -32,10 +32,11 @@
             "Str\\": "src/"
         }
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require-dev": {
-        "phpbench/phpbench": "1.0.*@dev",
```

Comment:
The `phpbench/phpbench:1.0-dev` version requires `php-7.2` version at least.

Using the `phpbench/phpbench:^0.16` to support `php-7.1` version.